src/trio/_timeouts.py

```diff
@@ -7,36 +7,38 @@
 import trio
 
 
-def move_on_at(deadline: float) -> trio.CancelScope:
+def move_on_at(deadline: float, *, shield: bool = False) -> trio.CancelScope:
     """Use as a context manager to create a cancel scope with the given
     absolute deadline.
 
     Args:
       deadline (float): The deadline.
+      shield (bool): Whether the cancel scope created is shielded.
```

Comment:
```suggestion
      shield (bool): Initial value for the `~trio.CancelScope.shield` attribute
          of the newly created cancel scope.
```
Suggest you link to the documentation for `CancelScope.shield` to clarify what shielding is about (in all four of these).